src/main/java/com/google/sps/data/Trip.java

```diff
@@ -0,0 +1,63 @@
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+
+/**
+ * Value class representing the data associated with a trip created on the My
+ * Trips page. Its corresponding Locations are stored in TripLocation entities.
+ */
+@AutoValue
+public abstract class Trip {
+
+  public static final String ENTITY_PROPERTY_TITLE = "title";
```

Comment:
Is the title the hotel name or address because I see you have a "hotel" variable as well?

Reply:
This is the name of the entity property for trip titles. The title is just the name of the trip, like "Amazing SF Trip"